_posts/2023-11-09-memories-from-crystalconf.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Memories from CrystalConf
+author: straight-shoota
+summary: Impressions from the conference in Berlin
+---
+
+CrystalConf 2023 was a sounding success with a terrific lineup of speakers and talks.
+We spent two great days in Berlin sharing knowledge, ideas, and beers.
+
+Recordings of the talks are available on YouTube.
+
+<a href="https://youtube.com/playlist?list=PLt-CsM4G1WoadONHl3zPN_Ts5PqH8TgMZ" class="btn btn-large btn-flat">Video playlist</a>
+
+<figure markdown="1">
+  ![](/assets/events/crystalconf2023/audience.jpg)
+  <caption>The audience listens to Johannes' talk about noticing little details that point towards deficiencies.</caption>
+</figure>
+
+<figure markdown="1">
+  ![](/assets/events/crystalconf2023/panel.jpg)
+  <caption>In the panel discussion we talked the future of Crystal with the audience. We also had the immense pleasure to share the stage with two core team alumni.</caption>
```

Comment:
Do we mention anything of Stephie being back?

Reply:
I simply shortened the caption.